Wikipedia/Code/NSDictionary+WMFExtensions.m

```diff
@@ -0,0 +1,13 @@
+
+#import "NSDictionary+WMFExtensions.h"
+
+@implementation NSDictionary (WMFExtensions)
+
+- (BOOL)containsNullObjects {
```

Comment:
@joewalsh I could go either way - you have a preference? Should we have 2 versions - one recursive and one not?